### PR TITLE
Create ephemeral staging environments

### PR DESCRIPTION
**Pull request by kkr16**

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      pull-requests: 'write'
+      id-token: 'write'
+
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Check out Github repository'
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: cd web ; npm install ; npm run build
```
Comment (enocom) [inline .github/workflows/pr.yaml:22]: These `;` should be `&&` just in case.

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    - uses: imjasonh/setup-ko@v0.4
+      name: 'Set up ko'
+      id: ko
+    - env:
+        KO_DOCKER_REPO: 'gcr.io/gcping-devrel'
+      name: 'Publish to Google Container Registry'
+      run: |-
+        gcloud auth configure-docker gcr.io --quiet
+        echo IMAGE=$(ko publish ./cmd/ping/) >> $GITHUB_ENV
```
Comment (enocom) [inline .github/workflows/pr.yaml:46]: Maybe I missed it, but are we setting `KO_DOCKER_REPO`?

**Review by enocom — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (kkr16) [reply .github/workflows/pr.yaml:46]: Yes:
https://github.com/GoogleCloudPlatform/gcping/blob/d3f5399cb87760f5bd39ecad9701fd6e6b4a8390/.github/workflows/pr.yaml#L41-L42

Comment (enocom) [reply .github/workflows/pr.yaml:46]: OK. Thanks!

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (kkr16) [reply .github/workflows/pr.yaml:22]: Good call will fix.

**Review by enocom — APPROVED** [review]
Just the semi-colon to ampersand fix. Otherwise LGTM

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (kkr16) [reply .github/workflows/pr.yaml]: Fixed:
https://github.com/GoogleCloudPlatform/gcping/blob/add-ghaction/.github/workflows/pr.yaml#L22

**Review by crwilcox — APPROVED** [review]

**Pull request merged**